Formula/field3d.rb

```diff
@@ -0,0 +1,30 @@
+class Field3d < Formula
+  desc "Library for storing voxel data on disk and in memory"
+  homepage "https://sites.google.com/site/field3d/"
+  url "https://github.com/imageworks/Field3D/archive/v1.7.2.tar.gz"
+  sha256 "8f7c33ecb4489ed626455cf3998d911a079b4f137f86814d9c37c5765bf4b020"
+
+  depends_on "scons" => :build
+  depends_on "boost"
+  depends_on "ilmbase"
+  depends_on "hdf5"
+
+  def install
+    scons
+    include.install Dir["install/**/**/release/include/*"]
```

Comment:
why aren't we using `scons "install"`?

Reply:
According to the [README](https://sites.google.com/site/field3d/building-field3d#TOC-Building-Field3D-on-Mac-OS-X), when using `scons` for building it just places the built files in an "install" directory, relying on a package manager (MacPorts in their example) to do the installing.